ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-eus-4.10.yaml

```diff
@@ -64,21 +60,37 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: aws-ipi-ovn-p2-f14
-  cron: 57 0 2,17 * *
+- as: vsphere-ipi-p2-f14
+  cron: 26 20 4,19 * *
+  remote_api: true
+  steps:
+    cluster_profile: vsphere
+    env:
+      E2E_RUN_INTERMEDIATE_TAGS: '@4.11 and @vsphere-ipi and @upgrade-sanity and not
+        @fips'
+      E2E_RUN_LATEST_TAGS: '@4.12 and @vsphere-ipi and @upgrade-sanity and not @fips'
+      UPGRADE_CHECK_RUN_TAGS: '@4.10 and @vsphere-ipi and @upgrade-check and not @fips'
+      UPGRADE_PRE_RUN_TAGS: '@4.10 and @vsphere-ipi and @upgrade-prepare and not @fips'
+    test:
+    - ref: cucushift-installer-check
+    - chain: openshift-upgrade-qe-test-paused
+    workflow: cucushift-installer-rehearse-vsphere-ipi
+- as: azure-ipi-p2-f14
```

Comment:
Could you help to sort the tests ?

Reply:
ah, thanks for pointing it out. Updated.